quizcon/templates/main/index.html

```diff
@@ -35,9 +35,20 @@ <h1 class="logo display-1" style="border: 0; font-weight: 700;">
             Built for creating confident choices<br />
             for testing in CourseWorks.
         </p>
-        <div style="margin: 2em 0 2em 0;">
-            <a href="/accounts/login/?next=/dashboard/" class="btn btn-lg btn-light" id="cy-login">Sign In</a>
-        </div>
+        {% if CAS_BASE %}
+
+            <form class="text-center" method="get" action="{{CAS_BASE}}cas/login">
+
+                    <input type="hidden" name="destination" value="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/accounts/caslogin/?next={{ next }}" />
+                    <div style="margin: 2em 0 2em 0;">
+                    <a href="" class="btn btn-lg btn-light" id="cy-login" data-cy="columbia-login" type="submit">
```

Comment:
Swap out the `a` tag for an `input` or `button`. Anchors aren't allowed the submit type.